Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/border.h

```diff
@@ -132,12 +130,10 @@ namespace Polygon_mesh_processing {
   * @tparam NamedParameters a sequence of \ref namedparameters
   *
   * @param pmesh the polygon mesh to which `faces` belong
-  * @param faces the range of faces defining the patch
-  *              around which the border is collected
-  * @param out the output iterator that collects halfedges that form the border
-  *            of `faces`, seen from inside the surface patch
-  * @param np optional sequence of \ref namedparameters among
-     the ones listed below
+  * @param faces the range of faces defining the patch whose border halfedges
+  *              are collected
+  * @param out the output iterator that collects the border halfedges of the patch
```

Comment:
can you please add "seen from outside"? If we have been confused, I think it's not overkill to repeat...